docs/maya/muscle.md

```diff
@@ -133,6 +135,7 @@ In order to provide more artistic control, some key parameters of the muscle sol
 | **Fibers**                      | {0.0, 0.0, 0.0} | The deformer estimates the fiber directions at each vertex based on the tendon weights. In case that the estimated fibers do not fit well to the desired directions, the paint tool can be used to comb the fibers manually. The fibers can be displayed using the *Muscle Fibers* option in the [debugger](#debugger). |
 | **Global Damping**              | 1.0             | Set global damping per vertex in the simulated mesh. The greater the value per vertex is the more it will attempt to retain its previous position. |
 | **Masses**                      | 1.0             | Set individual mass values per vertex in the simulated mesh. |
+| **Shape Preservation**           | 0.1             | Amount of correction to apply to the current vertex to maintain the initial state of the 3 vertices conforming a single shape preservation constraint. |
```

Comment:
```suggestion
| **Shape Preservation**          | 0.1             | Amount of correction to apply to the current vertex to maintain the initial shape state with the surrounding vertices. |
```

It is not that clear for me. I like this one more but also is not perfect. This comment might be applied to all the deformers.

Reply:
I see your point! I have rewritten this description